src/screens/authentication/utils/index.js

```diff
@@ -1,12 +1 @@
-import MessagingService from '~/messagingService'
-
 export const isValidEmail = (text) => text.includes('@')
-
-export const enableNotificationsForUser = async () => {
-  try{
-    await MessagingService.checkForPermissions()
-    await MessagingService.saveFCMTokenForCurrentUser()
-  } catch (error) {
-    console.error(error.message)
-  }
-}
```

Comment:
Was this old code that is not used anymore? Where are notifications enabled now?

Reply:
Yes, we abstracted it into the authentication process (emailSignUp, emailLogin and googleLogin)